### PR TITLE
Refactor wrapper functions

### PR DESCRIPTION
**Pull request by rhfogh**
Refactored/removed wrapper functions, and further renamed to new-style function names.

The changes are extensive enough that people will likely be a need to make some extra adjustments, but I cannot see another way of getting this done.

This PR does not test correct because the mockup objects need refactoring too  I should have that fixed before too long.

### CODE REVIEW

File: HardwareObjects/ALBA/ALBAFlux.py [hunk 1]
```diff
@@ -53,4 +53,4 @@ def get_dose_rate(self, energy=None):
 
 
 def test_hwo(hwo):
-    print(hwo.get_flux())
+    print(hwo.get_value)
```
Comment (IvarsKarpics) [inline HardwareObjects/ALBA/ALBAFlux.py]: get_value()

File: HardwareObjects/Resolution.py [hunk 2]
```diff
@@ -257,7 +240,7 @@ def update_energy(self, value):
         _wavelength = (h * c / e) / value * 10e6
 
         distance = self._hwr_detector.distance.get_value()
-        radius = self.get_detector_radius(distance)
+        radius = sself._hwr_detector.get_radius(distance)
```
Comment (marcus-oscarsson) [inline HardwareObjects/Resolution.py]: Watch out, typo ! sself

Comment (rhfogh) [reply HardwareObjects/Resolution.py]: Indeed. It is fixed in the next PR.
Thanks, Rasmus

**Comment by marcus-oscarsson** [comment]
Yes Indeed, really well done !, just one comment for the typo ... otherwise fine !

**Comment by marcus-oscarsson** [comment]
There is also a conflict ...

**Comment by rhfogh** [comment]
Yeah, I do not know what happened here. I am working on merging/rebasing on to the newest master, but it looks like it will need some manual checking before i can push that.

File: HardwareObjects/Resolution.py (hunk 2 above)
Comment (beteva) [reply HardwareObjects/Resolution.py]: Thanks @rhfogh for the huge work.
just one more  - _set value signature has changed
from  def _set_value(self, value, wait=False, timeout=None):
to def _set_value(self, value):

**Comment by rhfogh** [comment]
Thanks

**Pull request merged**